android/src/main/java/com/rnlib/adyen/AdyenPaymentModule.kt

```diff
@@ -178,10 +178,10 @@ class AdyenPaymentModule(private var reactContext : ReactApplicationContext) : R
                     val pmApiResponse : PaymentMethodsApiResponse = PaymentMethodsApiResponse.SERIALIZER.deserialize(JSONObject(response.body()?.string()))
                     val paymentMethodsList : MutableList<PaymentMethod> = mutableListOf<PaymentMethod>()
                     if(component != "dropin"){
-                        for (each in pmApiResponse.paymentMethods!!) {
-                            Log.i(TAG,each.toString())
-                            if (each.type == component) {
-                                paymentMethodsList.add(each)
+                        for (userPaymentMethod in pmApiResponse.paymentMethods!!) {
```

Comment:
Le's avoid the `!!` operator.
Please consider to use the following code as a more Kotlin-idiomatic way of convey the idea.

```
val  paymentMethods =  pmApiResponse.paymentMethods ?: emptyList()

for (userPaymentMethod in paymentMethods) {
     ...
}
```